cmd/helm/install.go

```diff
@@ -388,3 +397,19 @@ func defaultNamespace() string {
 	}
 	return "default"
 }
+
+func (i *installCmd) checkDependencies(ch *chart.Chart, reqs *chartutil.Requirements) {
+	deps := ch.GetDependencies()
```

Comment:
@technosophos Yeah, that's totally doable.  Let me throw that together real quick